docs/guides/PULL_REQUESTS.md

```diff
@@ -50,12 +50,12 @@ Before you submit your pull request consider the following guidelines:
 
   * Re-run the AngularJS Material test suite to ensure tests are still passing.
 
-  * Rebase your branch to **squash your commits** and **force push** to your GitHub repository 
+  * Commit your new changes by [ammending your original commit](https://help.github.com/articles/changing-a-commit-message/#amending-older-or-multiple-commit-messages) to **squash your commits**, then **force push** to your GitHub repository
```

Comment:
We actually don't need to require people to rebase / force push any more since we can now rebase + merge in GitHub.

Reply:
Actually I forgot that our script for syncing AngularJS Material into Google still needs commits this way, so it should stay.